docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -35,72 +35,15 @@ Note that the **Script** area works according to the selected language variable.
+This feature has been introduced as a component, so there is no need to use **Custom Script** for this task. For more details, see:
 
-Appcircle currently has `OpenJDK 11` (default), `OpenJDK 8`, `OpenJDK 17` and `OpenJDK 21`.
+- [Change Java Version](/workflows/common-workflow-steps/change-java-version)
 
-[Android Build](/workflows/android-specific-workflow-steps/android-build) step uses `OpenJDK 11` as default JDK version.
+However, if you prefer to update or improve it manually on **Custom Script**, the source code is available here:
 
-You can use the below custom script before your build step to change your `JAVA_HOME` environment variable.
-
-```bash
-echo "Default JAVA" $JAVA_HOME
-
-echo "OpenJDK 8" $JAVA_HOME_8_X64
-echo "OpenJDK 11" $JAVA_HOME_11_X64
-echo "OpenJDK 17" $JAVA_HOME_17_X64
-echo "OpenJDK 21" $JAVA_HOME_21_X64
-
-# Change JAVA_HOME to OPENJDK 17
-echo "JAVA_HOME=$JAVA_HOME_17_X64" >> $AC_ENV_FILE_PATH
-```
-
-Create a custom script like above and put it **above** your [Android Build](/workflows/android-specific-workflow-steps/android-build) step.
-
-<Screenshot url="https://cdn.appcircle.io/docs/assets/workflow-android-change-java-workflow.png" />
-
-<Screenshot url="https://cdn.appcircle.io/docs/assets/workflow-android-change-java-workflow-detail.png" />
-
-:::caution
-
-Please be aware that this custom script affects any step that comes after.
-
-Therefore, you should use this step as a standalone step instead of as part of any custom script.
-
-:::
-
-:::tip
-
-You can find more details about the included Java versions on the [Android Build Infrastructure](/infrastructure/android-build-infrastructure#java-version) page.
-
-:::
-
-:::info
-
-#### Changing System Java Version
-
-Changing the `JAVA_HOME` environment variable will be enough for your Android builds, but it won't change the `java` version in the system.
-
-If you're using a tool in the build pipeline that requires another Java version than the default OpenJDK 11, you should also change the system's default Java version using the below commands in the custom script.
-
-```bash
-source "$SDKMAN_DIR/bin/sdkman-init.sh"
-sdk default java $(basename $JAVA_HOME_17_X64)
-```
-
-After that, you will see the output of `java -version` as below in the build logs.
-
-```txt
-openjdk version "17.0.7" 2023-04-18 LTS
-OpenJDK Runtime Environment Zulu17.42+19-CA (build 17.0.7+7-LTS)
-OpenJDK 64-Bit Server VM Zulu17.42+19-CA (build 17.0.7+7-LTS, mixed mode, sharing)
-```
-
-You can also switch to other pre-installed Java versions using the relevant environment variable as an argument in the `sdk` command. For more details about these environment variables, see the [Android Build Infrastructure](/infrastructure/android-build-infrastructure#java-version) page.
-
-:::
+- [Change Java Version - Source Code](https://github.com/appcircleio/appcircle-change-java-version-component)
```

Comment:
@ftmzhrsckldr we can change the link with only github link. In this way, it can look like we give in the components source code.

for example: https://capture.dropbox.com/97kXqdO5GTdihygc

Reply:
@boztopuz I. did it this way on purpose. Because the other way is seem little complex but I think it is not necessary

Current:
![Screenshot 2024-10-28 at 12 29 36](https://github.com/user-attachments/assets/aed735d5-f488-4fc0-a70f-40925c00a5bb)

Feedback :
![Screenshot 2024-10-28 at 12 26 42](https://github.com/user-attachments/assets/37e60c65-ced9-4a40-a200-9571abb92430)


It is not on the component page for that IMO we can applied different format.

Do you think it is "must to"?

Reply:
Since we do source code redirection for all components in this way, I thought it should fit the current structure. It is not mandatory, I don't think it will cause a problem in its current form. @ftmzhrsckldr